### ISSUE TITLE
build sync issue In latest Android Studio: Unsupported method: BaseConfig.getApplicationIdSuffix()

### ISSUE THREAD

**Issue by truedat101** [DrJukka/WifiPairing#2]
Unsupported method: BaseConfig.getApplicationIdSuffix().
The version of Gradle you connect to does not support that method.
To resolve the problem you can change/upgrade the target version of Gradle you connect to.
Alternatively, you can ignore this exception and read other information from the model.

**Comment by truedat101** [comment]
Using Android Studio on os x: Android Studio 3.1.2
Build #AI-173.4720617, built on April 13, 2018
JRE: 1.8.0_152-release-1024-b01 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.6

**Comment by truedat101** [comment]
Note: while Android Studio build is broken (there must be a missing Gradle version dependency in the build settings), ./gradlew runs fine from the command line, and ./gradlew installDebug works.

**Comment by DrJukka** [comment]
Yep, android studio & gradle have changed greatly over the years. And unfortunately I only have time to update projects I would be working with, and quite many are not actively updated anymore
